mailbox/backup/src/main/java/org/apache/james/mailbox/backup/MailArchiveEntry.java

```diff
@@ -0,0 +1,7 @@
+package org.apache.james.mailbox.backup;
+
+/**
+ * Represents an element in an archive (Mailbox, MailboxAnnotation, Message)
```

Comment:
**instanceOf** expected here to know the entity?

Or is there a plan to have a generic MailArchiveEntry that knows how to restore itself?